examples/flutter_notifications/lib/notification/view/widgets/notification_list.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_notifications/notification/bloc/notification_bloc.dart';
+
+class NotificationList extends StatelessWidget {
+  const NotificationList({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<NotificationBloc, NotificationState>(
+      builder: (context, state) {
+        if (state is NotificationRecieveSuccess) {
+          return ListView.builder(
+              itemCount: 1,
```

Comment:
Are we only showing 1 notification at a time?

Reply:
Yes I initially only implemented 1 notification. multiple notifications can be implemented in future.